src/Routes.js

```diff
@@ -14,7 +14,7 @@ export const Routes = () => (
             <Route exact path='/' component={RosPage} />
             <Route path='/:inventoryId' component={RosSystemDetail} />
             <Route>
-                <Redirect to='/ros' />
+                <Redirect path="*" to='/' />
```

Comment:
We can omit the `Route` component which is nesting the `Redirect` component.
We can directly use the `Redirect`. Any thoughts? @PreetiW

Reply:
sure @r14chandra yes we can, I refactored it and pushed the changes. 💯